ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-commands.sh

```diff
@@ -94,9 +94,15 @@ oc wait --timeout=120m --for=condition=Available --namespace=clusters hostedclus
   exit 1
 }
 echo "Cluster became available, creating kubeconfig"
-bin/hypershift create kubeconfig --namespace=clusters --name=${CLUSTER_NAME} >${SHARED_DIR}/nested_kubeconfig || {
+KUBECONFIG_NAME=""
+while [[ -z "${KUBECONFIG_NAME}" ]]; do
```

Comment:
Ok I miss the entire context here but shouldn't this loop stops somehow if the kubeconfig doesn't show up at all?